miner/miner_optimism_test.go

```diff
@@ -45,9 +46,24 @@ func TestDAFootprintMining(t *testing.T) {
 			}
 			daFootprint += txs[i].RollupCostData().EstimatedDASize().Uint64() * testDAFootprintGasScalar
 		}
-		require.Less(t, txGas, block.GasUsed(), "total tx gas used must be smaller than block gas used")
-		require.Equal(t, daFootprint, block.GasUsed(), "total DA footprint used should be equal to block gas used")
+		require.Equal(t, txGas, block.GasUsed(), "total tx gas used should still be equal to block gas used")
```

Comment:
Nit, I think the "still" comes from historical context around the design of this feature and we don't need to commit it. 
```suggestion
		require.Equal(t, txGas, block.GasUsed(), "total tx gas used should be equal to block gas used")
```

Reply:
Should this function be named something like `requireDAFootprintGreaterThanBlockGasUsed`?

Reply:
I meant to use "still" because the behavior doesn't change. But I removed it, seems to confuse.

Reply:
renamed to `requireLargeDAFootprintBehavior`